plugins/cnfreg/config.go

```diff
@@ -36,14 +33,15 @@ type Config struct {
 	SwModHttpBasePort int `json:"sw-module-http-base-port"`
 	// The maximum amount of time StoneWork will wait for all CNFs (SW-Modules) to register themselves.
 	CnfDiscoveryTimeout time.Duration `json:"cnf-discovery-timeout"`
+	// Time interval between checks for StoneWork module registration.
+	CnfDiscoveryPollingRate time.Duration `json:"cnf-discovery-polling-rate"`
 }
 
 // loadConfig returns PuntMgr plugin file configuration if exists.
 func (p *Plugin) loadConfig() (*Config, error) {
 	cfg := &Config{
-		SwModGrpcBasePort:   defaultSwModGrpcBasePort,
-		SwModHttpBasePort:   defaultSwModHttpBasePort,
-		CnfDiscoveryTimeout: defaultSwCnfDiscoveryTimeout,
```

Comment:
I added a `CnfDiscoveryTimeout` deprecation warning as using it has no effect because of the changes in this PR. So users can still set it in `cnfreg.conf` but it will have no impact on StoneWork (except emitting the warning in logs).